pkgs/misc/vim-plugins/update.py

```diff
@@ -330,16 +331,32 @@ def generate_nix(plugins: List[Tuple[str, str, Plugin]]):
 });
 in lib.fix' (lib.extends overrides packages)
 """)
-    print("updated generated.nix")
+    print(f"updated {outfile}")
 
 
 def main() -> None:
-    plugin_names = load_plugin_spec()
+    parser = argparse.ArgumentParser(
+        description=("Updates nix derivations for vim plugins"
+            f"By default from {DEFAULT_IN} to {DEFAULT_OUT}"
+            )
+    )
+    parser.add_argument(
+        "--input-names", "-i", dest="input_file", default=DEFAULT_IN,
+        help="A list of plugins in the form owner/repo"
+    )
+    parser.add_argument(
+        "--out", "-o", dest="outfile", default=DEFAULT_OUT,
+        help="Filename to save generated nix code"
+    )
+
+    args = parser.parse_args()
```

Comment:
Can you put the argument parsing in a function `parse_args()` that return `args` int the end?

Reply:
done